src/martok/declarations/KlassGenerator.ts

```diff
@@ -189,35 +189,35 @@ export class KlassGenerator {
   ): ConstructorParameter {
     const name = node.name!.getText();
     const annotations: string[] = [];
-    let type = getMemberType(this.martok, node, {
+    const t = getMemberType(this.martok, node, {
```

Comment:
pretty short variable name, what's the motivation to shorten it like this?

Reply:
i added a property on that variable called `type` and didn't want to do `type.type` because that's gross. Open to changing

Reply:
I think `type.type` is the lesser evil compared to a single-letter name :-)

Reply:
i'm gonna go with `memberType.type` since that aligns more with the function naming